probe-rs/src/flashing/flasher.rs

```diff
@@ -201,7 +201,15 @@ impl<'session> Flasher<'session> {
                 })?;
             // We need to reload the flasher, since the debug sequence erase
             // may have invalidated any previously invalid state
-            self.load()
+            let result = self.load();
```

Comment:
The issue is that `StartFlashing` is not emitted, _not_ the `FinishedErasing` event